Sources/Fullscreen/AddressView/AddressView.swift

```diff
@@ -0,0 +1,222 @@
+        control.tintColor = .primaryBlue
+        return control
+    }()
+
+    private lazy var segmentContainer: UIView = {
+        let segmentContainer = UIView()
+        segmentContainer.translatesAutoresizingMaskIntoConstraints = false
+        segmentContainer.backgroundColor = .white
+        segmentContainer.addSubview(mapTypeSegmentControl)
+        segmentContainer.layer.masksToBounds = false
+        segmentContainer.layer.shadowOpacity = 0.3
+        segmentContainer.layer.shadowRadius = 3
+        segmentContainer.layer.shadowOffset = .zero
+        segmentContainer.layer.shadowColor = UIColor.black.cgColor
+
+        if UIDevice.isIPad() {
+            NSLayoutConstraint.activate([
+                mapTypeSegmentControl.widthAnchor.constraint(equalToConstant: 350),
+                mapTypeSegmentControl.centerXAnchor.constraint(equalTo: segmentContainer.centerXAnchor)
+                ])
+        } else {
+            NSLayoutConstraint.activate([
+                mapTypeSegmentControl.leadingAnchor.constraint(equalTo: segmentContainer.leadingAnchor, constant: .mediumLargeSpacing),
+                mapTypeSegmentControl.trailingAnchor.constraint(equalTo: segmentContainer.trailingAnchor, constant: -.mediumLargeSpacing)
+                ])
+        }
+
+        NSLayoutConstraint.activate([
+            mapTypeSegmentControl.topAnchor.constraint(equalTo: segmentContainer.topAnchor, constant: .mediumLargeSpacing),
+            mapTypeSegmentControl.bottomAnchor.constraint(equalTo: segmentContainer.bottomAnchor, constant: -.mediumLargeSpacing)
+            ])
+
+        return segmentContainer
+    }()
+
+    private lazy var mapView: MKMapView = {
+        let view = MKMapView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.delegate = self
+        return view
+    }()
+
+    private lazy var addressCardView: AddressCardView = {
+        let view = AddressCardView()
+        view.delegate = self
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var centerMapButton: UIButton = {
+        let button = UIButton(withAutoLayout: true)
+        button.backgroundColor = .milk
+        button.tintColor = .primaryBlue
+
+        button.layer.cornerRadius = 23
+        button.layer.shadowColor = UIColor.black.cgColor
+        button.layer.shadowOffset = CGSize(width: 0, height: 1)
+        button.layer.shadowRadius = 3
+        button.layer.shadowOpacity = 0.5
+
+        button.setImage(UIImage(named: .pin), for: .normal)
+        button.setImage(UIImage(named: .pin), for: .highlighted)
+        button.addTarget(self, action: #selector(centerMapButtonAction), for: .touchUpInside)
+
+        return button
+    }()
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        fatalError()
+    }
+
+    public var model: AddressViewModel? {
```

Comment:
Could you place this property at the top with the delegate? Just to have them grouped together

Reply:
Moved the delegate bellow so all the publics are together